tvb_documentation/manuals/UserGuide/UserGuide-UI_Simulator.rst

```diff
@@ -95,22 +95,17 @@ Configure a Simulation
 
 |
 
-A simulation is configured from the middle column.
+A simulation is configured in multiple steps in a wizard-like workflow.
 
 On the top of this column there is:
-  - a field to enter the new simulation name,
-  - the `Launch` button on the top right to start the simulation, and
-  - the `Configure Interface` button to select which of the simulation 
-    components are visible.
-  - links to pages that allow detailed configuration
-
-    - region model configuration
-    - region noise configuration
-    - surface level configuration (available for surface simulations)
+  - a dropdown field to choose a connectivity,
+  - a field to enter the conduction speed for the "Long-range connectivity",
+  - a dropdown field to choose the coupling function
+  - a Next button for the next step of the configuration
 
 |
 
-    .. image:: screenshots/simulator_configuration_buttons.jpg
```

Comment:
I think `simulator_configuration_buttons.jpg` should be removed. Or is it used elsewhere?